jwst/assign_wcs/pointing.py

```diff
@@ -223,10 +223,13 @@ def create_fitswcs(inp, input_frame=None):
         wcsinfo = wcsinfo_from_model(inp)
         wavetable = None
         spatial_axes, spectral_axes, unknown = gwutils.get_axes(wcsinfo)
-        sp_axis = spectral_axes[0]
-        if wcsinfo['CTYPE'][sp_axis] == 'WAVE-TAB':
-            wavetable = inp.wavetable
-        transform = fitswcs_transform_from_model(wcsinfo, wavetable)
+        if spectral_axes:
+            sp_axis = spectral_axes[0]
+            if wcsinfo['CTYPE'][sp_axis] == 'WAVE-TAB':
+                wavetable = inp.wavetable
+        else:
+            wavetable = None
```

Comment:
wavetable is defined already on line 224.

Reply:
Ah, I don't need the `else`.  Will fix.